### PR TITLE
Allow rendering of skipped locations in example app

### PR DESCRIPTION
**Pull request by owenpearson**
The new icons are from Google's material design icons and are free to use (https://google.github.io/material-design-icons/#licensing)

see this video if you don't wanna spend time setting up the example app locally
https://user-images.githubusercontent.com/48608556/140076930-e0a56bb3-5d3e-4aa2-89e0-a6738a60e8b0.mov

### CODE REVIEW

File: examples/subscribing-example-app/public/RiderConnection.js [hunk 1]
```diff
@@ -52,6 +55,15 @@ export class RiderConnection {
     this.subscriber.start(channelId || 'ivan');
   }
 
+  setRenderSkippedLocations(state) {
+    this.renderSkippedLocations = state;
+  }
+
+  setSkippedLocationInterval(interval) {
+    console.log(`setting skipped location interval to ${interval}`);
```
Comment (QuintinWillison) [inline examples/subscribing-example-app/public/RiderConnection.js:63]: In particular for browser runtime, I'm always a bit cautious of any kind of console logging - especially when it's using raw APIs which can't be effectively disabled at runtime without removing code.

I notice we have [a `Logger` class defined](https://github.com/ably/ably-asset-tracking-js/blob/c35753121c46464cd0b3b694d0ec5b1d09b630da/src/lib/utils/Logger.ts#L13) in this codebase. Is there a reason you cannot use that here?

On [a quick search](https://github.com/ably/ably-asset-tracking-js/search?q=console.log), it doesn't seem that we use `console.log` elsewhere right now in this app.

File: examples/subscribing-example-app/public/RiderConnection.js [hunk 2]
```diff
@@ -64,7 +76,22 @@ export class RiderConnection {
       marker.focus();
     }
 
-    this.rider.move(locationCoordinate, this.shouldSnap);
+    if (this.timeouts.length) {
```
Comment (QuintinWillison) [inline examples/subscribing-example-app/public/RiderConnection.js:79]: Coercion to boolean here is a bit icky (if you want to be specific, then `> 0` would be more readable)... though I think this check is probably superfluous if `this.timeouts` is always defined.

**Review by QuintinWillison — APPROVED** [review]
This looks good to go, as is, so feel free to address my comments by creating issues rather than fixing in this pull request.

File: examples/subscribing-example-app/public/RiderConnection.js (hunk 1 above)
Comment (owenpearson) [reply examples/subscribing-example-app/public/RiderConnection.js]: My bad, I was just using this for local debugging, definitely didn't mean to push it up. d200f5d.

File: examples/subscribing-example-app/public/RiderConnection.js (hunk 2 above)
Comment (owenpearson) [reply examples/subscribing-example-app/public/RiderConnection.js]: df4e743

**Pull request merged**